ddl/owner_manager.go

```diff
@@ -134,25 +139,25 @@ func newSession(ctx goctx.Context, flag string, etcdCli *clientv3.Client, retryC
 
 // CampaignOwner implements OwnerManager.CampaignOwner interface.
 func (m *ownerManager) CampaignOwner(ctx goctx.Context) error {
-	ddlSession, err := newSession(ctx, DDLOwnerKey, m.etcdCli, newSessionDefaultRetryCnt, ManagerSessionTTL)
+	session, err := newSession(ctx, m.prompt, m.key, m.etcdCli, newSessionDefaultRetryCnt, ManagerSessionTTL)
 	if err != nil {
 		return errors.Trace(err)
 	}
-	ddlCtx, _ := goctx.WithCancel(ctx)
-	go m.campaignLoop(ddlCtx, ddlSession, DDLOwnerKey)
+	cancelCtx, _ := goctx.WithCancel(ctx)
+	go m.campaignLoop(cancelCtx, session)
 	return nil
 }
 
-func (m *ownerManager) campaignLoop(ctx goctx.Context, etcdSession *concurrency.Session, key string) {
-	idInfo := fmt.Sprintf("%s ownerManager %s", key, m.ddlID)
+func (m *ownerManager) campaignLoop(ctx goctx.Context, etcdSession *concurrency.Session) {
+	idInfo := fmt.Sprintf("%s ownerManager %s", m.key, m.id)
 	var err error
 	for {
 		select {
 		case <-etcdSession.Done():
-			log.Infof("[ddl] %s etcd session is done, creates a new one", idInfo)
-			etcdSession, err = newSession(ctx, idInfo, m.etcdCli, newSessionRetryUnlimited, ManagerSessionTTL)
+			log.Infof("[%s] %s etcd session is done, creates a new one", m.prompt, idInfo)
+			etcdSession, err = newSession(ctx, m.prompt, idInfo, m.etcdCli, newSessionRetryUnlimited, ManagerSessionTTL)
```

Comment:
Can we use a log prefix instead of `m.prompt` and `idInfo` here?  We can pass an argument.